force-app/main/default/classes/ContactController.cls

```diff
@@ -31,4 +39,22 @@ public with sharing class ContactController {
             LIMIT 1
         ];
     }
+
+    @AuraEnabled
+    public static string updateContacts(Object data) {
+        List<Contact> contactsForUpdate = (List<Contact>) JSON.deserialize(
+            JSON.serialize(data),
+            List<Contact>.class
+        );
+        try {
+            // Make sure we can update the database before even trying
+            if (!Schema.sObjectType.Contact.isUpdateable()) {
+                return null;
```

Comment:
We need to throw a user friendly error here, not silently abort the operation.

Reply:
Done!